tests/scripts/test_PMID_downloader.py

```diff
@@ -0,0 +1,143 @@
+import os
+from unittest import mock
+
+import pytest
+from click.testing import CliRunner
+from docling.document_converter import DocumentConverter
+
+from scripts.PMID_downloader import PMID_downloader
+import pathlib
+import tempfile
+
+CI = bool(os.getenv("GITHUB_ACTIONS"))
+
+
+@pytest.mark.skipif(CI, reason="CI needs internet access for this test")
+def test_PMID_downloader(request):
+
+    # The first PMID of this dummy_pmids.txt file should give a PDF
+    # The second PMID of the file should not provide a PDF since there is no corresponding PMCID
+    dummy_pmids_file_path = str(
+        pathlib.Path(request.path).parent.parent / "assets/scripts/dummy_pmids.txt"
+    )
+    with open(dummy_pmids_file_path, "r") as file:
+        dummy_pmids: list[str] = file.readlines()
+
+    runner = CliRunner()
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        result = runner.invoke(PMID_downloader, [dummy_pmids_file_path, tmpdirname])
+
+        assert (
+            result.exit_code == 0
+        ), f"CLI exited with code {result.exit_code}: {result.output}"
+
+        PDF_file_names = [f for f in os.listdir(tmpdirname)]
+        PDF_file_names_no_file_type = [f.split(".")[0] for f in os.listdir(tmpdirname)]
+        dummy_PMID_names = [line.split("_")[1].rstrip() for line in dummy_pmids]
```

Comment:
Guess, these are the expected pmid names?

```suggestion
        expected_PMID_names = [line.split("_")[1].rstrip() for line in dummy_pmids]
```